packages/storybook/src/stories/grid-rendering.stories.tsx

```diff
@@ -40,3 +52,28 @@ export const RenderInputInCell = () => {
     </div>
   );
 };
+export const InfiniteLoading = () => {
+  const { loading, data, setRowLength } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 50,
+    maxColumns: 20,
+  });
+
+  const handleOnRowsScrollEnd = (params) => {
+    const newRowLength = params.virtualRowsCount + params.viewportPageSize;
+    setRowLength(newRowLength);
```

Comment:
Could we add a max number of rows,  so we can check how the behaviour is when the server returns all its rows.

Reply:
Sure, do you have a specific number in mind? I can make it max out at 500 for example.

Reply:
Perfect

Reply:
Updated